GVRf/Framework/framework/src/main/java/org/gearvrf/GVRAndroidResource.java

```diff
@@ -313,6 +313,23 @@ public synchronized final InputStream getStream() throws IOException{
         return stream;
     }
 
+    /**
```

Comment:
Why not make ``GVRAndroidResource(String name, InputStream inputStream)`` public or a variant of it? This method circumvents all the existing ctors since you can use any of them and then override the stream with this method. It completely disables the encapsulation GVRAndroidResource is trying to do.